### PR TITLE
Mayaqua/CMakeLists: Fix win32 build without vcpkg

### PR DESCRIPTION
**Pull request by domosekai**
Changes proposed in this pull request:
 - Add `ws2_32.lib` and `crypt32.lib` to `CMakeLists.txt` to fix linking issues
 - 
 -

### CODE REVIEW

**Review by pravitva — COMMENTED** [review]
- [ ] - - -

File: src/Mayaqua/CMakeLists.txt [hunk 1]
```diff
@@ -63,6 +63,8 @@ if(WIN32)
       "setupapi.lib"
       "winmm.lib"
       "WtsApi32.Lib"
+      "ws2_32.lib"
+      "crypt32.lib"
```
Comment (davidebeatrici) [inline src/Mayaqua/CMakeLists.txt:67]: The libraries are actually in alphabetical order, I would keep it.

Comment (domosekai) [reply src/Mayaqua/CMakeLists.txt]: OK

**Review by pravitva — COMMENTED** [review]
Show me code

**Pull request merged**